src/idl_gen_python.cpp

```diff
@@ -141,9 +141,17 @@ static void GetScalarFieldOfTable(const StructDef &struct_def,
   code += MakeCamel(field.name);
   code += "(self):";
   code += OffsetPrefix(field);
-  code += Indent + Indent + Indent + "return " + getter;
-  code += "o + self._tab.Pos)\n";
-  code += Indent + Indent + "return " + field.value.constant + "\n\n";
+  getter += "o + self._tab.Pos)";
+  if (field.value.type.base_type == BASE_TYPE_BOOL) {
```

Comment:
this fixes the case for reading fields, but what about for writing (serializing) them?

Reply:
I think serializing is fine, don't know what to improve.

Reply:
Ah seems bools are already handled there.